package/MDAnalysis/analysis/density.py

```diff
@@ -394,8 +395,15 @@ def _prepare(self):
                 logger.warning(msg)
             # Generate a copy of smin/smax from coords to later check if the
             # defined box might be too small for the selection
-            smin = np.min(coord, axis=0)
-            smax = np.max(coord, axis=0)
+            try: 
```

Comment:
I could be wrong but I don't see how `np.min` can ever be `False` and therefore trigger the below `ValueError` unless `self._atomgroup.positions` is empty. Although this is the desired behaviour, IMO this is not the best way to test it. 

@IAlibay is there a canonical way of checking for an empty `atomgroup`?

Reply:
Which way would you prefer -- an `if len(coord)`? I actually prefer ``try/except`` here because the case where there are no atoms seems much rarer than the opposite, but I haven't put much thought into this.

Reply:
AGs can coerce to bool, so empty is false. Lily is right wrt EAFP

Reply:
Okay sure, I agree and on second look the error message details what it is meant to be checking so all good.